### PR TITLE
Resolve extension enabled status when loading it

### LINKED ISSUE #1471: Extensions are silently disabled on Lens restart
After quitting Lens and restarting, extensions that were enabled are no longer enabled. Happening in master and 4.0.0-beta.4

### PR DESCRIPTION
**Pull request by nevalla**
Fixes #1471 

Signed-off-by: Lauri Nevala <lauri.nevala@gmail.com>

### CODE REVIEW

File: src/extensions/extensions-store.ts [hunk 1]
```diff
@@ -34,11 +34,6 @@ export class ExtensionsStore extends BaseStore<LensExtensionsStoreModel> {
     await extensionLoader.whenLoaded;
     await this.whenLoaded;
 
-    // activate user-extensions when state is ready
```
Comment (nevalla) [inline src/extensions/extensions-store.ts:50]: extensionManager has not loaded extensions at this point so we cannot determine the status here.

**Review by Nokel81 — APPROVED** [review]

**Comment by nevalla** [comment]
This is conflicted with https://github.com/lensapp/lens/pull/1482, so let's re-implement this after it is merged.

File: src/extensions/extension-discovery.ts [hunk 2]
```diff
@@ -213,24 +214,31 @@ export class ExtensionDiscovery {
...
 
     try {
       // check manifest file for existence
       fs.accessSync(manifestPath, fs.constants.F_OK);
 
       manifestJson = __non_webpack_require__(manifestPath);
+      const installedManifestPath = path.join(this.nodeModulesPath, manifestJson.name, "package.json");
       this.packagesJson.dependencies[manifestJson.name] = path.dirname(manifestPath);
 
+      if (!isBundled) {
```
Comment (panuhorsmalahti) [inline src/extensions/extension-discovery.ts]: [nit] mutation could be avoided: `const isEnabled = isBundled || extensionsStore.isEnabled(installedManifestPath);`

File: src/extensions/extensions-store.ts (hunk 1 above)
Comment (ixrock) [reply src/extensions/extensions-store.ts:50]: Should we add above something like `await extensionManager.whenReady`?

**Review by panuhorsmalahti — APPROVED** [review]

File: src/extensions/extension-discovery.ts (hunk 2 above)
Comment (nevalla) [reply src/extensions/extension-discovery.ts]: 👍 will change

**Pull request merged**